### ISSUE TITLE
RTC audio output stops after playing sound on iOS

### ISSUE THREAD

**Issue by optimygmbh** [flutter-webrtc/flutter-webrtc#275]
**Describe the bug**
If I play a sound with package assets_audio_player or audioplayers, the phone call audio stops.

**To Reproduce**
Create a RTC call and play a sound with package: assets_audio_player and function: 
`AssetsAudioPlayer.playAndForget(Audio('assets/background.mp3'));`

**Expected behavior**
The phone call sound should still be there, same like it is working on Android.

**Platform information**

* **Flutter version**:
[✓] Flutter (Channel master, 1.19.0-2.0.pre, on Mac OS X 10.15.5 19F101, locale en-CH)
    • Flutter version 1.19.0-2.0.pre at /Users/myllertym/flutter
    • Framework revision 1d395c5e18 (8 days ago), 2020-05-31 07:41:50 -0700
    • Engine revision 923687f0e7
    • Dart version 2.9.0 (build 2.9.0-11.0.dev 6489a0c68d)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/myllertym/Library/Android/sdk/
    • Platform android-29, build-tools 28.0.3
    • ANDROID_HOME = /Users/myllertym/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.8.4

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 43.0.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.45.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.11.0

* **Plugin version**:
0.2.8
* **OS**: 
iOS
* **OS version**:
13

**Comment by azamshahriar** [comment]
I am facing same problem. When my app is running at that time sf some one phone call.
Then audio focus change-> my other app audio sound become too low?? 
Plz suggest me?

**Comment by ycherniavskyi** [comment]
On iOS this behavior is controlled with [`AVAudioSession`](https://developer.apple.com/documentation/avfoundation/avaudiosession).
First, you need to check the correctness of used options of the audio session’s category which set by [`setCategory:withOptions:error:`](https://developer.apple.com/documentation/avfoundation/avaudiosession/1616442-setcategory?language=objc).
And then the correctness and of [`setActive:error:`](https://developer.apple.com/documentation/avfoundation/avaudiosession/1616597-setactive) calls.

**Comment by optimygmbh** [comment]
Thanks for you answer.
I checked the code of the `audioplayers ` package. What I saw is this:

[The category of AVAudioSession is set here](https://github.com/luanpotter/audioplayers/blob/master/darwin/Classes/AudioplayersPlugin.m#L455-L480)

I saw the category on `flutter-webrtc` is set to `AVAudioSessionCategoryPlayAndRecord` according to the [code](https://github.com/cloudwebrtc/flutter-webrtc/blob/master/ios/Classes/FlutterWebRTCPlugin.m#L493-L494). 

What I don't understand: what is now the right setting? Or where is the bug? On this package or on the `audioplayers` package?

Thanks for your help in advance! If I understand what's wrong I can do it and send a pull request.
